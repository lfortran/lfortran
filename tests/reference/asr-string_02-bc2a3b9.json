{
    "basename": "asr-string_02-bc2a3b9",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/string_02.f90",
    "infile_hash": "249fc849c605e3aad0bd2c8a33e00523fdc2270b95caa69737801caa",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-string_02-bc2a3b9.stdout",
    "stdout_hash": "8f86447b63a892b5429bc28d79cf70d050b7318cf212f6444b75ed8c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}