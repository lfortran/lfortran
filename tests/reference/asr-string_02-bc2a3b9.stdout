(TranslationUnit
    (SymbolTable
        1
        {
            string_02:
                (Program
                    (SymbolTable
                        2
                        {
                            firstname:
                                (Variable
                                    2
                                    firstname
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Character 1 15 ())
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            greetings:
                                (Variable
                                    2
                                    greetings
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Character 1 25 ())
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            surname:
                                (Variable
                                    2
                                    surname
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Character 1 15 ())
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            title:
                                (Variable
                                    2
                                    title
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Character 1 6 ())
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    string_02
                    []
                    [(Assignment
                        (Var 2 title)
                        (StringConstant
                            "Mr. "
                            (Character 1 4 ())
                        )
                        ()
                    )
                    (Assignment
                        (Var 2 firstname)
                        (StringConstant
                            "Rowan "
                            (Character 1 6 ())
                        )
                        ()
                    )
                    (Assignment
                        (Var 2 surname)
                        (StringConstant
                            "Atkinson"
                            (Character 1 8 ())
                        )
                        ()
                    )
                    (Assignment
                        (Var 2 greetings)
                        (StringConstant
                            "A big hello from Mr. Bean"
                            (Character 1 25 ())
                        )
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "Here is "
                                (Character 1 8 ())
                            )
                            (Var 2 title)
                            (Var 2 firstname)
                            (Var 2 surname)]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(Var 2 greetings)]
                        ()
                        ()
                    )]
                )
        })
    []
)
